### ISSUE TITLE
Attach debugger to existing process

### ISSUE THREAD

**Issue by DonJayamanne** [DonJayamanne/pythonVSCode#330]
Add support for attaching the debugger to an existing process

**Comment by DonJayamanne** [comment]
Unfortunately, this might take a lot longer (a few weeks) than initially anticipated.

**Comment by pankajp** [comment]
@DonJayamanne Did you start work on it? Do you have a WIP branch or so on your work?
https://github.com/fabioz/PyDev.Debugger has code to attach to processess on mac, linux and windows. I suppose the code from it to attach to existing process can be extracted to be useful in general. I guess the only issues might be interfacing with the native code and maybe license compatibility.

**Comment by DonJayamanne** [comment]
@pankajp
My plan was to use the pydev.debugger. However there didn't seem to be much interest on this feature. No likes (voted) on this, hence this was never looked AT&T and fell off the radar.

I would still like to use this, which would mean I would have to change everything in the debugger. Also, it has the added benefit of debugging django apps with live reloadong. That's a definite plus, however not much interest from the dev community.

In short, my plan is to work in this slowly, no era. Would be mid next year-end as it is a big refactor/rewrite.

**Comment by DonJayamanne** [comment]
If you would like to contribute I could start a branch and lay the ground work.

**Comment by pankajp** [comment]
@DonJayamanne While I am very much interested in this feature, I don't think I would have the time to work on it much given my inexperience in working with VSCode and its language server/debugger protocol and the plenty of free time available after $dayjob.

**Comment by DonJayamanne** [comment]
Closing for now

**Comment by fdanny** [comment]
Any chance this was implemented? I was trying to figure out how to attach to a process.

**Comment by RockyTV** [comment]
So, is there any chance of implementing live reload for Django apps?
